03_launch_mgmt_cluster.sh

```diff
@@ -59,6 +59,23 @@ function launch_minikube() {
     minikube start
 }
 
+function launch_kinder() {
+  kinder="${GOPATH}/bin/kinder"
+  if [[ $(sudo $kinder get clusters) == "" ]]; then
+    sudo $kinder create cluster
+    sudo $kinder do kubeadm-init
```

Comment:
the cluster won't have an interface in the provisioning network, that is problematic if for example we move ironic inside the cluster